source/blog/2017-06-02-rpm-ostree-v2017.6-released.html.md

```diff
@@ -0,0 +1,220 @@
+---
+title: rpm-ostree v2017.6 released
+author: jlebon
+date: 2017-06-02 18:00:00 UTC
+tags: rpm-ostree, rpm, ostree, atomic, atomic host
+comments: true
+published: true
+---
+
+[rpm-ostree](https://rpm-ostree.readthedocs.org/) is the
+hybrid image/package system that provides transactional
+upgrades on Atomic Host. Here are some highlights from
+version
+[v2017.6](https://github.com/projectatomic/rpm-ostree/releases/tag/v2017.5).
+
+READMORE
+
+The highlight of this release is experimental support for
+live system updates. You can test it out by
+[rebasing your Fedora Atomic Host](https://fedoraproject.org/wiki/QA:Updates_Testing#Using_it_with_Atomic_Host_.28Fedora_24_and_later.29)
+onto the testing branch. Feel free to also leave karma in
+the pending Bodhi updates:
+
+- [Fedora 25 update](https://bodhi.fedoraproject.org/updates/FEDORA-2017-85014ae3f6)
+- [Fedora 26 update](https://bodhi.fedoraproject.org/updates/FEDORA-2017-f86a299577)
+
+### Live updates
+
+rpm-ostree is now capable of applying some updates live
+without the need to reboot using the `rpm-ostree ex livefs`
+command. This works only in situations where the only
```

Comment:
I do say it at the end of the section, though I guess it doesn't hurt to make it more obvious.